app/helpers/field_validation_helper.rb

```diff
@@ -93,6 +95,11 @@ def validate_email_address(field, email_address)
     end
   end
 
+  def email_address_suggestion(email_address)
+    suggestion = Mailcheck.new.suggest(email_address)
```

Comment:
We should guard against `email_address` being nil by returning early, since Mailcheck will throw a `NoMethodError` if `email_address` is nil.

```
NoMethodError
undefined method `downcase' for nil:NilClass
/app/vendor/bundle/ruby/2.6.0/gems/mailcheck-1.0.0/lib/mailcheck.rb:17:in `suggest'
/app/app/helpers/field_validation_helper.rb:99:in `email_address_suggestion'
/app/app/controllers/coronavirus_form/check_contact_details_controller.rb:6
```

This can happen if you visit `/check-contact-details` without an email address in your contact details hash.

Reply:
Thanks @bilbof! I've updated and fixed up.